fabtests/common/shared.c

```diff
@@ -4165,9 +4169,25 @@ struct option long_opts[] = {
 	{"pin-core", required_argument, NULL, LONG_OPT_PIN_CORE},
 	{"timeout", required_argument, NULL, LONG_OPT_TIMEOUT},
 	{"debug-assert", no_argument, &debug_assert, LONG_OPT_DEBUG_ASSERT},
+	{"data-progress-model", required_argument, NULL, LONG_OPT_DATA_PROGRESS},
+	{"control-progress-model", required_argument, NULL, LONG_OPT_CONTROL_PROGRESS},
```

Comment:
These feel like a mouthful. Not sure if there's a much better alternative, maybe shorten to --data_progress=manual / auto / unified?